x-pack/plugins/event_log/server/types.ts

```diff
@@ -31,6 +41,18 @@ export interface IEventLogService {
   getLogger(properties: IEvent): IEventLogger;
 }
 
+export interface IEventLogClientService {
+  getClient(request: KibanaRequest): IEventLogClient;
+}
+
+export interface IEventLogClient {
+  findEventsBySavedObject(
+    type: string,
+    id: string,
+    options?: Partial<FindOptionsType>
+  ): Promise<IEvent[]>;
+}
+
```

Comment:
This approach (rather than extending the existing EventLogService) was in order to align this with the AlertClient which is also exposed on the `Start` api and extends the context of a request.